src/isolate.rs

```diff
@@ -176,11 +187,30 @@ impl Isolate {
     });
     let config = libdeno::deno_config {
       will_snapshot: 0,
-      load_snapshot: snapshot,
+      load_snapshot: match init.snapshot {
+        Some(s) => s,
+        None => libdeno::deno_buf::empty(),
+      },
       shared: libdeno::deno_buf::empty(), // TODO Use for message passing.
       recv_cb: pre_dispatch,
     };
     let libdeno_isolate = unsafe { libdeno::deno_new(config) };
+    // Run init script if present.
+    match init.init_script {
+      Some(init_script) => {
+        let filename_cstr = CString::new(init_script.filename).unwrap();
+        let source_cstr = CString::new(init_script.source).unwrap();
+        unsafe {
+          libdeno::deno_execute(
+            libdeno_isolate,
+            null(),
+            filename_cstr.as_ptr(),
+            source_cstr.as_ptr(),
+          );
```

Comment:
Why not use Isolate::execute2 instead of adding another call to `libdeno::deno_execute`? I'm concerned there might be (now or in the future) extra bits added to `Isolate::execute` which then would not be executed in this call?

Reply:
Would doing this after creating the isolate struct be fine?

Reply:
If that works for cross compiling and could simplify the IsolateInit arg - that would be preferable.